contrib/mpd/files/mpd

```diff
@@ -0,0 +1,9 @@
+# MPD service
+
+type            = process
+command         = /usr/bin/mpd --no-daemon
+before          = network.target
```

Comment:
why is it before network? seems like it should be before login.target if anything